llvm/utils/lit/tests/Inputs/shtest-cat/cat-e.txt

```diff
@@ -0,0 +1,33 @@
+## Tests to check cat -e output for ASCII characters and newlines
+
+# RUN: cat -e %S/txt-files/allchars | FileCheck %s --check-prefix=ALLCHARS
+# RUN: cat -e %S/txt-files/newline | FileCheck %s --check-prefix=NEWLINE
+
+# ALLCHARS: ^@^A^B^C^D^E^F^G^H      $
```

Comment:
```suggestion
# ALLCHARS: ^@^A^B^C^D^E^F^G^H
# ALLCHARS-SAME: $
```
May want to exclude chars that we can't output/see in the FileCheck line.`{{.*}}$` could also work.

My concern is those just look like whitespace, but are probably something else.

Reply:
done